### PR TITLE
vendor: Bump github.com/cilium/hubble

### PR DESCRIPTION
**Pull request by gandro**
This pulls in the latest version of [github.com/cilium/hubble](https://github.com/cilium/hubble). Adds support for policy verdict events in Hubble.

### CODE REVIEW

**Comment by gandro** [comment]
test-me-please

File: go.sum [hunk 1]
```diff
@@ -106,6 +106,8 @@ github.com/cilium/ebpf v0.0.0-20191113100448-d9fb101ca1fb/go.mod h1:MA5e5Lr8slmE
 github.com/cilium/hubble v0.0.0-20200304205543-565cbe60f7fb/go.mod h1:GYdTLWzzpzgp2ptyEp0LccPotGo4BxPX9SJRGL4pI5c=
 github.com/cilium/hubble v0.0.0-20200320161706-5faac2a81350 h1:Emt6IBafrgi95wUCKJdspH6vuTx9mmpwOEEwo61j5U4=
```
Comment (tklauser) [inline go.sum]: I'd suggest to run `go mod tidy` to clean stray entries from `go.sum`.

Comment (gandro) [reply go.sum]: Good catch! Done.

**Comment by gandro** [comment]
test-me-please

**Review by michi-covalent — APPROVED** [review]

**Review by christarazi — APPROVED** [review]

**Pull request merged**